environments/prod/prod.tfvars

```diff
@@ -1607,6 +1607,34 @@ frontends = [
         ]
       },
     ],
+    disabled_rules = {
+      SQLI = [
```

Comment:
exclusions are preferred over disabled rules,

given there was none previously I assume the team has tried to avoid disabling them.

This needs the owning team to review it. (again we do not usually get involved in this, other than to push back on overly broad suprressions)